examples/examples.py

```diff
@@ -170,8 +170,16 @@ def stft_plot(signal, sample_rate=DEFAULT_SAMPLE_RATE):
 
 # Envelope test
 adsr = ADSR(
-    attack=a, decay=d, sustain=s, release=r, alpha=alpha, synthglobals=synthglobals
+    attack=a,
+    decay=d,
+    sustain=s,
+    release=r,
+    alpha=alpha,
+    synthglobals=synthglobals,
+    device=device,
 ).to(device)
+# I'm curious here if just doing device=device is enough and we
```

Comment:
@jorshi thoughts? I kinda want it set up so you never really need to do `.to` if you pass in the device.